src/Protocol/DFRN.php

```diff
@@ -1500,15 +1500,20 @@ private static function fetchauthor($xpath, $context, $importer, $element, $only
 
 		$fields = ['id', 'uid', 'url', 'network', 'avatar-date', 'avatar', 'name-date', 'uri-date', 'addr',
 			'name', 'nick', 'about', 'location', 'keywords', 'xmpp', 'bdyear', 'bd', 'hidden', 'contact-type'];
-		$condition = ["`uid` = ? AND `nurl` = ? AND `network` != ? AND NOT `pending` AND NOT `blocked` AND `rel` IN (?, ?)",
-			$importer["importer_uid"], Strings::normaliseLink($author["link"]), Protocol::STATUSNET,
-			Contact::SHARING, Contact::FRIEND];
+		$condition = ["`uid` = ? AND `nurl` = ? AND `network` != ? AND NOT `pending` AND NOT `blocked`",
+			$importer["importer_uid"], Strings::normaliseLink($author["link"]), Protocol::STATUSNET];
+
+		if ($importer['account-type'] != User::ACCOUNT_TYPE_COMMUNITY) {
+			$condition = DBA::mergeConditions($condition, ['rel' => [Contact::SHARING, Contact::FRIEND]]);
+		}
+
 		$contact_old = DBA::selectFirst('contact', $fields, $condition);
 
 		if (DBA::isResult($contact_old)) {
 			$author["contact-id"] = $contact_old["id"];
 			$author["network"] = $contact_old["network"];
 		} else {
+			Logger::info('Blubb', ['condition' => $condition]);
```

Comment:
Please remove this debug call.

Reply:
Örgs :-)

Reply:
"blubb" in fact is an abreviation for ... ehemm, ... let me look ... 😁

Reply:
Done